release/example/str.trie_sencoten/example.str.trie_sencoten.model_info

```diff
@@ -2,7 +2,8 @@
     "name": "Example SENĆOŦEN Trie Model",
     "license": "mit",
     "languages": [
-        "str"
+        "str",
+        "str-Latn"
```

Comment:
Languages will be sourced from the .kps if not included here. Nearly all fields in .model_info are sourced from .kps -- see https://help.keyman.com/developer/cloud/model_info/1.0/

The intent is that only metadata needed for distribution on the website is included in the source .model_info. The destination .model_info is built from that and from the .kps data. This should make it pretty DRY

Reply:
Ah! Somehow I didn't realize all of that from the table. 😖 Is the best course of action then to explicitly provide just the license and the description in the `.model_info` files?